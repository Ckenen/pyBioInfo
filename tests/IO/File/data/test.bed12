chr1	12251	12389	E00552:229:HN2YKCCXY:6:2109:8379:63156	3.0	+	12251	12251	0,0,255	1	138,	0,
chr1	14743	14998	E00552:229:HN2YKCCXY:6:1217:13859:50392	1.0	-	14743	14743	0,0,255	2	86,29,	0,226,
chr1	16627	16726	E00552:229:HN2YKCCXY:6:1116:18406:31969	0.0	+	16627	16627	0,0,255	1	99,	0,
chr1	17295	17565	E00552:229:HN2YKCCXY:6:2110:8734:1538	1.0	+	17295	17295	0,0,255	2	73,40,	0,230,
chr1	17957	18059	E00552:229:HN2YKCCXY:6:2221:16701:66918	3.0	-	17957	17957	0,0,255	1	102,	0,
chr1	131629	131731	E00552:229:HN2YKCCXY:6:1213:26829:62066	0.0	-	131629	131629	0,0,255	1	102,	0,
chr1	135709	135853	E00552:229:HN2YKCCXY:6:1204:17594:5985	0.0	+	135709	135709	0,0,255	1	144,	0,
chr1	136843	137220	E00552:229:HN2YKCCXY:6:2119:30736:64122	0.0	-	136843	136843	0,0,255	2	11,73,	0,304,
chr1	138162	138249	E00552:229:HN2YKCCXY:6:2216:23236:43958	0.0	-	138162	138162	0,0,255	1	87,	0,
chr1	185220	186333	E00552:229:HN2YKCCXY:6:1216:21349:48248	0.0	+	185220	185220	0,0,255	2	130,17,	0,1096,
chr1	186325	186408	E00552:229:HN2YKCCXY:6:1213:15706:37946	0.0	-	186325	186325	0,0,255	1	83,	0,
chr1	188503	188847	E00552:229:HN2YKCCXY:6:1104:8024:72701	1.0	-	188503	188503	0,0,255	2	81,57,	0,287,
chr1	188831	195340	E00552:229:HN2YKCCXY:6:2117:6421:27029	0.0	-	188831	188831	0,0,255	2	71,78,	0,6431,
chr1	377413	377478	E00552:229:HN2YKCCXY:6:1105:6745:67111	0.0	-	377413	377413	0,0,255	1	65,	0,
chr1	509676	751449	E00552:229:HN2YKCCXY:6:2213:25743:68342	255.0	-	509676	509676	0,0,255	2	43,61,	0,241712,
chr1	629314	629421	E00552:229:HN2YKCCXY:6:2120:1509:29912	3.0	-	629314	629314	0,0,255	1	107,	0,
chr1	629662	629751	E00552:229:HN2YKCCXY:6:2216:18862:45734	3.0	+	629662	629662	0,0,255	1	89,	0,
chr1	629695	629820	E00552:229:HN2YKCCXY:6:2124:2037:49320	3.0	-	629695	629695	0,0,255	1	125,	0,
chr1	629766	629837	E00552:229:HN2YKCCXY:6:2108:30766:15338	3.0	-	629766	629766	0,0,255	1	71,	0,
chr1	629873	630018	E00552:229:HN2YKCCXY:6:1210:6816:66250	3.0	-	629873	629873	0,0,255	1	145,	0,
chr1	629967	630019	E00552:229:HN2YKCCXY:6:1123:20090:47298	3.0	+	629967	629967	0,0,255	1	52,	0,
chr1	630322	630407	E00552:229:HN2YKCCXY:6:1106:20202:71154	3.0	+	630322	630322	0,0,255	1	85,	0,
chr1	630323	630432	E00552:229:HN2YKCCXY:6:1106:17239:57512	3.0	-	630323	630323	0,0,255	1	109,	0,
chr1	630323	630416	E00552:229:HN2YKCCXY:6:1213:6502:40073	3.0	+	630323	630323	0,0,255	1	93,	0,
chr1	630332	630459	E00552:229:HN2YKCCXY:6:1210:5223:20348	3.0	+	630332	630332	0,0,255	1	127,	0,
chr1	630340	630408	E00552:229:HN2YKCCXY:6:1108:20750:48230	3.0	+	630340	630340	0,0,255	1	68,	0,
chr1	630368	630409	E00552:229:HN2YKCCXY:6:1203:2280:21614	3.0	+	630368	630368	0,0,255	1	41,	0,
chr1	630412	630478	E00552:229:HN2YKCCXY:6:2119:6847:66091	3.0	+	630412	630412	0,0,255	1	66,	0,
chr1	630773	630876	E00552:229:HN2YKCCXY:6:1215:5041:65529	3.0	+	630773	630773	0,0,255	1	103,	0,
chr1	630846	630983	E00552:229:HN2YKCCXY:6:2222:22810:61398	3.0	-	630846	630846	0,0,255	1	137,	0,
chr1	631199	631250	E00552:229:HN2YKCCXY:6:1213:23815:69185	3.0	-	631199	631199	0,0,255	1	51,	0,
chr1	631222	631326	E00552:229:HN2YKCCXY:6:1203:10450:52291	3.0	-	631222	631222	0,0,255	1	104,	0,
chr1	631245	631377	E00552:229:HN2YKCCXY:6:1119:22455:44257	3.0	-	631245	631245	0,0,255	1	132,	0,
chr1	631270	631357	E00552:229:HN2YKCCXY:6:1207:9090:45857	3.0	-	631270	631270	0,0,255	1	87,	0,
chr1	631902	631991	E00552:229:HN2YKCCXY:6:2204:25144:9501	3.0	-	631902	631902	0,0,255	1	89,	0,
chr1	632044	632098	E00552:229:HN2YKCCXY:6:2105:28706:30474	3.0	-	632044	632044	0,0,255	1	54,	0,
chr1	632148	632248	E00552:229:HN2YKCCXY:6:2218:17432:20084	3.0	+	632148	632148	0,0,255	1	100,	0,
chr1	632490	632599	E00552:229:HN2YKCCXY:6:1223:6177:2628	3.0	-	632490	632490	0,0,255	1	109,	0,
chr1	632527	632628	E00552:229:HN2YKCCXY:6:2203:3934:60940	3.0	-	632527	632527	0,0,255	1	101,	0,
chr1	632531	632641	E00552:229:HN2YKCCXY:6:2118:20222:58180	3.0	-	632531	632531	0,0,255	1	110,	0,
chr1	632540	632600	E00552:229:HN2YKCCXY:6:1113:17959:44169	3.0	-	632540	632540	0,0,255	1	60,	0,
chr1	632545	632669	E00552:229:HN2YKCCXY:6:1219:16894:2417	3.0	-	632545	632545	0,0,255	1	124,	0,
chr1	632547	632640	E00552:229:HN2YKCCXY:6:1116:20476:11787	3.0	-	632547	632547	0,0,255	1	93,	0,
chr1	633908	634003	E00552:229:HN2YKCCXY:6:1101:6340:43730	3.0	-	633908	633908	0,0,255	1	95,	0,
chr1	633946	634072	E00552:229:HN2YKCCXY:6:1124:28371:28136	255.0	-	633946	633946	0,0,255	1	126,	0,
chr1	633953	634083	E00552:229:HN2YKCCXY:6:1207:8937:65880	255.0	+	633953	633953	0,0,255	1	130,	0,
chr1	633956	634100	E00552:229:HN2YKCCXY:6:1223:25855:55561	255.0	-	633956	633956	0,0,255	1	144,	0,
chr1	633964	634066	E00552:229:HN2YKCCXY:6:2104:25662:8429	255.0	+	633964	633964	0,0,255	1	102,	0,
chr1	633983	634091	E00552:229:HN2YKCCXY:6:2109:31111:10064	255.0	+	633983	633983	0,0,255	1	108,	0,
chr1	633991	634104	E00552:229:HN2YKCCXY:6:1217:13261:13633	255.0	+	633991	633991	0,0,255	1	113,	0,
chr1	633993	634079	E00552:229:HN2YKCCXY:6:2201:6461:28857	255.0	+	633993	633993	0,0,255	1	86,	0,
chr1	634005	634111	E00552:229:HN2YKCCXY:6:2216:23236:10978	255.0	-	634005	634005	0,0,255	1	106,	0,
chr1	634043	634094	E00552:229:HN2YKCCXY:6:1111:26727:37911	3.0	+	634043	634043	0,0,255	1	51,	0,
chr1	634126	634193	E00552:229:HN2YKCCXY:6:1203:21440:1538	3.0	-	634126	634126	0,0,255	1	67,	0,
chr1	634127	634210	E00552:229:HN2YKCCXY:6:1105:30868:54506	3.0	+	634127	634127	0,0,255	1	83,	0,
chr1	634159	634227	E00552:229:HN2YKCCXY:6:2218:26646:4579	3.0	+	634159	634159	0,0,255	1	68,	0,
chr1	634425	634492	E00552:229:HN2YKCCXY:6:2119:17229:67937	3.0	+	634425	634425	0,0,255	1	67,	0,
chr1	733082	733180	E00552:229:HN2YKCCXY:6:1111:4208:49109	0.0	+	733082	733082	0,0,255	1	98,	0,
chr1	769580	769663	E00552:229:HN2YKCCXY:6:2121:12347:71471	3.0	+	769580	769580	0,0,255	1	83,	0,
chr1	827588	827683	E00552:229:HN2YKCCXY:6:1107:14813:1942	255.0	+	827588	827588	0,0,255	1	95,	0,
chr1	908219	908325	E00552:229:HN2YKCCXY:6:2202:2270:21772	255.0	-	908219	908219	0,0,255	1	106,	0,
chr1	910057	910125	E00552:229:HN2YKCCXY:6:1108:5244:49215	255.0	+	910057	910057	0,0,255	1	68,	0,
chr1	915746	915861	E00552:229:HN2YKCCXY:6:1206:31943:70891	255.0	-	915746	915746	0,0,255	1	115,	0,
chr1	916556	916673	E00552:229:HN2YKCCXY:6:1107:8653:65740	255.0	-	916556	916556	0,0,255	1	117,	0,
chr1	917756	917891	E00552:229:HN2YKCCXY:6:1208:21369:4614	255.0	+	917756	917756	0,0,255	1	135,	0,
chr1	919145	919295	E00552:229:HN2YKCCXY:6:1102:26819:42745	255.0	-	919145	919145	0,0,255	1	150,	0,
chr1	921145	921265	E00552:229:HN2YKCCXY:6:1101:22069:21860	255.0	-	921145	921145	0,0,255	1	120,	0,
chr1	924183	924333	E00552:229:HN2YKCCXY:6:1223:29995:2012	255.0	+	924183	924183	0,0,255	1	150,	0,
chr1	924597	924711	E00552:229:HN2YKCCXY:6:1115:31426:45066	255.0	+	924597	924597	0,0,255	1	114,	0,
chr1	924851	924949	E00552:229:HN2YKCCXY:6:2102:27245:32162	255.0	+	924851	924851	0,0,255	1	98,	0,
chr1	925503	925962	E00552:229:HN2YKCCXY:6:1212:26849:39458	255.0	+	925503	925503	0,0,255	2	21,41,	0,418,
chr1	925524	925638	E00552:229:HN2YKCCXY:6:2211:20598:3419	255.0	+	925524	925524	0,0,255	1	114,	0,
chr1	926884	927034	E00552:229:HN2YKCCXY:6:2222:21359:1151	255.0	+	926884	926884	0,0,255	1	150,	0,
chr1	928321	928418	E00552:229:HN2YKCCXY:6:2219:19999:36065	255.0	+	928321	928321	0,0,255	1	97,	0,
chr1	930279	935790	E00552:229:HN2YKCCXY:6:1214:16752:35783	255.0	-	930279	930279	0,0,255	3	57,51,19,	0,759,5492,
chr1	939065	939321	E00552:229:HN2YKCCXY:6:1101:13261:42042	255.0	+	939065	939065	0,0,255	2	64,50,	0,206,
chr1	939574	939639	E00552:229:HN2YKCCXY:6:2211:31324:25024	255.0	-	939574	939574	0,0,255	1	65,	0,
chr1	941294	942425	E00552:229:HN2YKCCXY:6:1214:3407:59358	255.0	-	941294	941294	0,0,255	3	12,116,16,	0,841,1115,
chr1	942458	942659	E00552:229:HN2YKCCXY:6:1206:9780:47158	255.0	-	942458	942458	0,0,255	2	30,101,	0,100,
chr1	942509	942659	E00552:229:HN2YKCCXY:6:1111:26098:6267	255.0	-	942509	942509	0,0,255	1	150,	0,
chr1	942743	942877	E00552:229:HN2YKCCXY:6:1120:7740:57583	255.0	+	942743	942743	0,0,255	1	134,	0,
