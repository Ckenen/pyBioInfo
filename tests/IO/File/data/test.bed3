chr1	12251	12389
chr1	14743	14998
chr1	16627	16726
chr1	17295	17565
chr1	17957	18059
chr1	131629	131731
chr1	135709	135853
chr1	136843	137220
chr1	138162	138249
chr1	185220	186333
chr1	186325	186408
chr1	188503	188847
chr1	188831	195340
chr1	377413	377478
chr1	509676	751449
chr1	629314	629421
chr1	629662	629751
chr1	629695	629820
chr1	629766	629837
chr1	629873	630018
chr1	629967	630019
chr1	630322	630407
chr1	630323	630432
chr1	630323	630416
chr1	630332	630459
chr1	630340	630408
chr1	630368	630409
chr1	630412	630478
chr1	630773	630876
chr1	630846	630983
chr1	631199	631250
chr1	631222	631326
chr1	631245	631377
chr1	631270	631357
chr1	631902	631991
chr1	632044	632098
chr1	632148	632248
chr1	632490	632599
chr1	632527	632628
chr1	632531	632641
chr1	632540	632600
chr1	632545	632669
chr1	632547	632640
chr1	633908	634003
chr1	633946	634072
chr1	633953	634083
chr1	633956	634100
chr1	633964	634066
chr1	633983	634091
chr1	633991	634104
chr1	633993	634079
chr1	634005	634111
chr1	634043	634094
chr1	634126	634193
chr1	634127	634210
chr1	634159	634227
chr1	634425	634492
chr1	733082	733180
chr1	769580	769663
chr1	827588	827683
chr1	908219	908325
chr1	910057	910125
chr1	915746	915861
chr1	916556	916673
chr1	917756	917891
chr1	919145	919295
chr1	921145	921265
chr1	924183	924333
chr1	924597	924711
chr1	924851	924949
chr1	925503	925962
chr1	925524	925638
chr1	926884	927034
chr1	928321	928418
chr1	930279	935790
chr1	939065	939321
chr1	939574	939639
chr1	941294	942425
chr1	942458	942659
chr1	942509	942659
chr1	942743	942877
