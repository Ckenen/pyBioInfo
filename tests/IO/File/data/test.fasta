>seq1
TTCCTGAAGAACATAATGCCTGCCTACCATCAACAGGCATCAATACTTTCTACCAGCTAT
TCTCAACCCTCATGATCGGAAGAGACAGAGACTGACTGTGTCAAAGTATTAGTCCCATCA
TTCAGCAATTAACTTTAGCTCAATGCTTCAAAAATTCTTCAGGCCCTGTGTAATTTCAGC
TACGTACATTAATGATGAGTACCCATACAACC
>seq2
ATTCTGTTTCTTATTTTCAGTACCATAT
>seq3
TTAATAAATATCAGTTATTCAATACTTTATTTAGACATTTTGTTAGATTATTTTGACCAA
CTGAAGTCTAATCTAAATGTTCTGAGCATGTTCAAAGTAAGCTAGGCCAACCTATAATTT
TCGGTGTGCTAAATGCATTTTTAACTTATGATATTTTCAGTTTACGGGGGTTTGTTGAGA
CATAACTTCATCATACATCAAGGAGCATCTGTATATGGGATATAGTTAAAGCAGTGATCA
GAGGAAAATCTATAGCCTTAACACATTTATTAATAAAAGTGTAGGAATTAAATTATCAGC
TGAAAAATGTAAAAAGTATCTAAAAGAGTAAGCAGAAAGTACAAGAAAGAACCCAAAGTA
GAAAAAAGTGAAAATTAATAAAATAAGAAGCCAAAAAACAGATCAAATCAGTAAACCAAA
AATCTTGTTCTTTAAACAAATCAACAAAGTTGACAAAAAAATTAGATCTTTTAATCATGA
ATAAAAAAAAGAGAAAGCACAAAAATGAATAAGGAATGGTGAGAGAAATAACTATTGATA
ATCAGCAAATAAAAAATCATTAAAAACAATGTTGTTCACATCTATGAAAAACATTGAAAG
CTAGAGGGAATGGGTAATTTTCTAGAAAAATACAATTCACCACAACTGACTTCAAAAAAA
AAAAAAAAAAAAAAGAAGTACCGCACTTATGTGAGCAATTTCCATAGAGAAATACAGTTG
TCATGGAATTATAACACACACACAAACACTAGGTTTAGATGTTTTCACAGAGAATTCCAC
CAAACCTTTAGAAATCAGATCGTCCAAAGGCAAATTAAC
