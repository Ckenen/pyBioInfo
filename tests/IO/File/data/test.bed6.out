chr1	11868	14409	ENST00000456328	.	+
chr1	11871	14412	ENST00000515242	.	+
chr1	11873	14409	ENST00000518655	.	+
chr1	12009	13670	ENST00000450305	.	+
chr1	14362	24886	ENST00000541675	.	-
chr1	14362	29370	ENST00000423562	.	-
chr1	14362	29370	ENST00000438504	.	-
chr1	14403	29570	ENST00000488147	.	-
chr2	38813	46385	ENST00000327669	.	-
chr2	41219	42952	ENST00000460464	.	-
chr2	41220	46870	ENST00000461026	.	-
chr2	197568	202605	ENST00000437798	.	+
chr2	217729	218866	ENST00000460974	.	-
chr2	218135	264032	ENST00000403712	.	-
chr2	218137	242800	ENST00000415006	.	-
chr2	218148	264392	ENST00000463865	.	-
chr2	218150	261323	ENST00000403657	.	-
chr2	218152	262786	ENST00000473104	.	-
